src/FirstSteps/RPG/Heroes/HeroesDamage.cs

```diff
@@ -0,0 +1,28 @@
+﻿
+using System;
+
+
+namespace FirstSteps.RPG.Heroes
+{
+    public class HeroesDamage
+    {
+
+        public void Damage(Hero hero, Boss boss)
```

Comment:
I think that fighting logic could be placed in the Hell class. Especially if you try to solve the task with the interface that I describe above.